chameleon.py

```diff
@@ -68,20 +90,28 @@ def stop():
 def _get_cherrypy_page_handler(status_code, content_type, content):
     class Foo(object):
         @cherrypy.expose
+        @cherrypy.tools.json_out()
```

Comment:
ok.  If you think it's good this way after looking I will pull and fiddle with it afterwards to satisfy my own curiosity.